project/install.sh

```diff
@@ -125,14 +125,21 @@ case "$lsb_dist" in
 
 		# aufs is preferred over devicemapper; try to ensure the driver is available.
 		if ! grep -q aufs /proc/filesystems && ! $sh_c 'modprobe aufs'; then
-			kern_extras="linux-image-extra-$(uname -r)"
+			if echo $(uname -r) | grep -q '\-generic' && dpkg -l linux-image-*-generic | grep '^ii' > /dev/null 2>&1; then
```

Comment:
Hmm, indeed.  Not sure what exactly I was doing with the echo there but yes, that's great, thanks for catching that!